bind/doc.go

```diff
@@ -0,0 +1,11 @@
+// Copyright 2017 The go-python Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+/*
+bind package generates language bindings that make it possible
+to call Go code and pass objects from Python.
+
+*/
+
```

Comment:
remove that empty line (otherwise the documentation isn't attached to the package)